src/NServiceBus.Core/Routing/RoutingOptionExtensions.cs

```diff
@@ -32,13 +33,56 @@ public static void OverrideReplyToAddressOfIncomingMessage(this ReplyOptions opt
         }
 
         /// <summary>
-        /// Routes this message to the local endpoint instance.
+        /// Routes this message to any instance of this endpoint.
+        /// </summary>
+        /// <param name="option">Context being extended.</param>
+        public static void RouteToThisEndpoint(this SendOptions option)
+        {
+            option.Context.GetOrCreate<UnicastSendRouterConnector.State>()
+                .Option = UnicastSendRouterConnector.RouteOption.RouteToAnyInstanceOfThisEndpoint;
+        }
+
+        /// <summary>
+        /// Routes this message to this endpoint instance.
         /// </summary>
         /// <param name="option">Context being extended.</param>
-        public static void RouteToLocalEndpointInstance(this SendOptions option)
+        public static void RouteToThisInstance(this SendOptions option)
```

Comment:
Endpoint instances are part of our concepts overview (http://docs.particular.net/nservicebus/concept-overview) and also `SendOptions` is kind of advanced API. The "beginner" API on `IBusContext/Session` (SendLocal) uses `RouteToThisEndpoint`.